onnx/defs/schema.h

```diff
@@ -25,6 +25,40 @@
 #include "onnx/onnx-operators_pb.h"
 namespace ONNX_NAMESPACE {
 
+struct FunctionBodyQueryContext {
+  virtual const AttributeProto* getAttribute(const std::string& name) const = 0;
+  virtual size_t getNumInputs() const = 0;
+  virtual size_t getNumOutputs() const = 0;
+  virtual ~FunctionBodyQueryContext() {}
+};
+
+struct FunctionBodyQueryContextImpl : public FunctionBodyQueryContext {
```

Comment:
are you going to have different implementation in ORT?

Reply:
yes.